src/layouts/Header/components/Navigator/Navigator.tsx

```diff
@@ -38,6 +42,15 @@ function Navigator({ currentUser }: NavigatorProps) {
     )
   }
 
+  if (ownerData && !ownerData.isCurrentUserPartOfOrg) {
```

Comment:
could consolidate this to !ownerData?.isCurrentUserPartOfOrg to accomplish the same thing

Reply:
In this case that doesn't work as that condition is true when ownerData is undefined, which we don't want.